pom.xml

```diff
@@ -181,21 +181,26 @@
           </execution>
         </executions>
       </plugin>
-<!--
-      See https://jira.apache.org/jira/browse/HUDI-304
+
+<!--  See https://jira.apache.org/jira/browse/HUDI-304-->
       <plugin>
         <groupId>com.diffplug.spotless</groupId>
```

Comment:
this plugin has some licensing issues before, not sure if it is ok to bring back now cc @bvaradar

Reply:
Thanks for reviewing. spotless and scalafmt both using Apache 2.0 License so I think it will be fine. Maybe it was on the Java side?